askama_escape/src/lib.rs

```diff
@@ -150,6 +150,57 @@ impl Escaper for Text {
     }
 }
 
+pub struct Latex;
+
+impl Escaper for Latex {
+    fn write_escaped<W>(&self, mut fmt: W, string: &str) -> fmt::Result
+    where
+        W: Write,
+    {
+        let mut last = 0;
+        for (index, byte) in string.bytes().enumerate() {
+            const MIN_CHAR: u8 = b'"';
+            const MAX_CHAR: u8 = b'~';
+            const TABLE: [Option<&&str>; (MAX_CHAR - MIN_CHAR + 1) as usize] = {
+                let mut table = [None; (b'~' - b'"' + 1) as usize];
+                table[(b'"' - MIN_CHAR) as usize] = Some(&r#"{\textquotedbl}"#);
```

Comment:
```suggestion
            const TABLE: [Option<&str>; (MAX_CHAR - MIN_CHAR + 1) as usize] = {
                let mut table = [None; (b'~' - b'"' + 1) as usize];
                table[(b'"' - MIN_CHAR) as usize] = Some(r#"{\textquotedbl}"#);
```

I assume it might be an artifact from copying and pasting another escaper. But is there a reason for `&&str` instead of just `&str`?